app/assets/javascripts/application.js

```diff
@@ -14,7 +14,6 @@
 //= require rails-ujs
 //= require bootstrap.growl
 //= require activestorage
-//= require turbolinks
```

Comment:
Can we keep turbolinks? It improves speed!

Reply:
sure